src/renderer/components/ExportLogsButton.js

```diff
@@ -99,7 +99,7 @@ const ExportLogsBtn = ({ hookToShortcut, primary = true, small = true, title, ..
   return hookToShortcut ? (
     <KeyHandler keyValue="e" onKeyHandle={onKeyHandle} />
   ) : (
-    <Button small={small} primary={primary} event="ExportLogs" onClick={handleExportLogs} {...rest}>
+    <Button small={small} primary={primary} event="Save logs" onClick={handleExportLogs} {...rest}>
```

Comment:
is this intentional? changing analytics name have a cost for data team because there is overlap during the event rename over users updating.

Reply:
I'll have to confirm with Chris and Ben regarding that.
What i understood is that they wanted to rename to standardize a bit more the event naming.
But yeah doing so will imply overlap between old and new versions.